README.md

```diff
@@ -8,6 +8,7 @@ Blogging my baking expeditions. #bakingexpeditions #noms
     cd baking-expeditions
     npm install # Install dependencies from package.json
     webpack # Generate JS bundle
+    gulp sass # Build CSS
     npm start
 
```

Comment:
It might even be possible to do the SASS compilation in gulp, but then do bundling in webpack.

I would just want to be with consistent. For example, right now my typescript is being compiled to JS and then bundled all in webpack. Perhaps I could move the typescript compilation out of webpack to gulp, and then use webpack for bundling. I don't know what sourcemaps are though, and they seem to be important in that process somewhere. Whatever they are, both gulp and webpack have support for them.